julia/1.julia-setup.md

```diff
@@ -97,9 +97,9 @@ Platform Info:
   WORD_SIZE: 64
   LIBM: libopenlibm
   LLVM: libLLVM-14.0.6 (ORCJIT, apple-m1)
-  Threads: 4 on 6 virtual cores
+  Threads: 1 on 6 virtual cores
 Environment:
-  JULIA_NUM_THREADS = 4
+  JULIA_NUM_THREADS = 1
   JULIA_PROJECT = @.
   JULIA_PKG_SERVER = https://mirror.iscas.ac.cn/julia
```

Comment:
JULIA_PKG_SERVER is not consistent with the above information.

Reply:
This behavior is expected. In the tutorial, I only setup a one time `JULIA_PKG_SERVER` to install `PkgClientServer` package. after that `JULIA_PKG_SERVER` will be managed automatically by `PkgClientServer`. In my case, `iscas.ac.cn` server has lowest lag so it's automatically chosen by `PkgClientServer`

Reply:
This is completely misleading. You tell readers to setup `JULIA_PKG_SERVER` and use `PkgClientServer` to waste the effort.

Reply:
> This is completely misleading. You tell readers to setup `JULIA_PKG_SERVER` and use `PkgClientServer` to waste the effort.

i told them to setup `JULIA_PKG_SERVER` for a one time only use by just pasting ` export JULIA_PKG_SERVER=...` into the terminal because if theu don't do that they cant even get PkgClientServer. this avoids having to modify `JULIA_PKG_SERVER` in startup.jl should  the connection to one mirror server become bad.